src/bridge/cockpitrouter.c

```diff
@@ -785,7 +819,7 @@ cockpit_router_init (CockpitRouter *self)
 
   /* Owns the channels */
   self->channels = g_hash_table_new_full (g_str_hash, g_str_equal, NULL, object_unref_if_not_null);
-  self->groups = g_hash_table_new_full (g_str_hash, g_str_equal, g_free, g_free);
+  self->groups = g_hash_table_new_full (g_str_hash, g_str_equal, g_free, channel_group_unref);
   self->fences = g_hash_table_new_full (g_str_hash, g_str_equal, g_free, NULL);
 
   /* The rules, including a default */
```

Comment:
Ah, this is probably hard to see in the github history: This referred to "bridge: Allow sharing objects between channels", and my question is to test the new `cockpit_channel_get_group_object()` API; particularly, add an object to channel1, see if you get the same on channel2, close channel1, ensure it's still on channel2, close channel2, ensure it gets invalidated -- something along that line?